### PR TITLE
Added my screens.

### PR DESCRIPTION
**Pull request by easyaspi314**
S2 Skyrocket
MacBook Mid-2009 (MacBook5,2)

### CODE REVIEW

File: screens.json [hunk 1]
```diff
@@ -643,6 +649,13 @@
 		"ppi": 217
 	},
 	{
+		"name": "Samsung Galaxy S2 Skyrocket"
+		"w": 480,
```
Comment (LeaVerou) [inline screens.json:653]: What is wrong with the colon here?

Comment (maxcmiller) [inline screens.json:653]: Looks like a missing comma at the end of the line above the colon (line 652).

**Comment by LeaVerou** [comment]
Hi there,
Please merge conflicts and add the missing comma? Thanks!

**Comment by LeaVerou** [comment]
Closed via #180

**Pull request closed without merging**